sources/ingest/fetch/location.py

```diff
@@ -67,6 +67,11 @@ class OntologyObject:
     id: str
     label: LanguageString
 
+@dataclass
+class Image:
+    image_url: str
+    caption: LanguageString
```

Comment:
Why is one field prefixed with `image` and the other is not?

`image.image_url`
`image.image_caption`

`image.url`
`image.caption`

Reply:
Updated to last option, thanks for raising this up.